app/components/programme_status_component/programme_status_component.html.erb

```diff
@@ -0,0 +1,20 @@
+
+<%= render GovGridRowComponent.new(additional_classes: "programme-status-wrapper", background_color: "white") do |row| %>
+  <%= row.with_column("two-thirds") do %>
+    <div class="programme-status-wrapper__title-row">
+      <h2 class="govuk-heading-m"><%= title %></h2>
+      <% if @user_programme_enrolment.in_state?(:complete) %>
+        <%= image_pack_tag "media/images/icons/programme_completed_icon.svg" %>
+      <% end %>
+      </div>
+    <%= render content.render %>
+  <% end %>
+  <%= row.with_column("one-third") do %>
+    <% if @user_programme_enrolment.in_state?(:complete) %>
+      <div class="programme-status-wrapper__button-container">
+        <%= link_to 'Download your certificate', @user_programme_enrolment.programme.certificate_path, 
+          class: 'govuk-button button', role: 'button', draggable: 'false' %></p>
```

Comment:
Do we need the draggable: false? I copied the same link_to in a previous PR and I think we decided this wasn't needed?

Reply:
Good spot, no its not, I will take that out